Algorithms/Numeric/KrishnamurthyNumberChecker.cs

```diff
@@ -0,0 +1,38 @@
+using System;
+
+namespace Algorithms.Numeric
+{
+    /// <summary>
+    /// A Krishnamurthy number is a number whose sum of the factorial of digits
+    /// is equal to the number itself.
+    ///
+    /// For example, 145 is a Krishnamurthy number since: 1! + 4! + 5! = 1 + 24 + 120 = 145.
+    /// </summary>
+    public static class KrishnamurthyNumberChecker
+    {
+        /// <summary>
+        /// Check if a number is Krishnamurthy number or not.
+        /// </summary>
+        /// <param name="n">The number to check.</param>
+        /// <returns>True if the number is Krishnamurthy, false otherwise.</returns>
+        public static bool IsKMurthyNumber(int n)
```

Comment:
Could you make it work with long? Or better BigInteger if possible. Factorials do get quite large quite quickly!

Reply:
True, factorials can grow quite large, but if you notice it only calculates the factorial of each digit of the number.
Thus the maximum factorial it will calculate is at most 9! or 362880. We are just fine with integer here :)